go.mod

```diff
@@ -1,15 +1,16 @@
 module github.com/GoogleContainerTools/kaniko
 
-go 1.17
+go 1.19
 
 require (
 	cloud.google.com/go/storage v1.29.0
 	github.com/Azure/azure-storage-blob-go v0.14.0
+	github.com/GoogleCloudPlatform/docker-credential-gcr v1.5.1-0.20210909233216-4cdd60d0f2d8
```

Comment:
This maintains the same version of `docker-credential-gcr` as was in the `Dockerfile` above.  
The version of `docker-credential-acr-env` below also matcheshe `Dockerfile`.